src/main/java/interface_adapters/EventController.java

```diff
@@ -47,7 +48,7 @@ public EventController(CalendarEventCreationBoundary eventAdder, EventScheduler
     public boolean createEvent(String eventName, LocalDateTime startDateTime, Duration duration) {
         // todo use exceptions to ensure that duration won't last until the next day
         LocalTime endTime = startDateTime.plus(duration.getSeconds(), ChronoUnit.SECONDS).toLocalTime();
-        return createEvent(eventName, startDateTime.toLocalTime(), endTime, new HashSet<String>(), startDateTime.toLocalDate());
+        return createEvent(eventName, startDateTime.toLocalTime(), endTime, new HashSet<>(), startDateTime.toLocalDate());
```

Comment:
Not going to ask you to change this, but going to keep adding reminders to keep unrelated changes to a minimum. If it's a small refactoring change I can review it quickly and get it accepted faster than this big PR.

Reply:
The same applies to the CalendarEventModel modifier changes and EventGetter dependency changes, which one of us will probably get merge conflict issues because I'm removing those methods entirely in the Event dependency inversion PR.